pom.xml

```diff
@@ -286,6 +287,7 @@
                                 <useSubDirectoryPerArtifact>true</useSubDirectoryPerArtifact>
                                 <stripVersion>true</stripVersion>
                                 <excludeScope>provided</excludeScope>
+                                <excludes>home/**</excludes>
```

Comment:
Perhaps we can state `<excludes>/**</excludes>` instead? The real problem was the presence of zip entries with an absolute path.